src/class-parsely.php

```diff
@@ -129,6 +131,52 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	/**
+	 * Include a link to the statistics page for an article in the wp-admin Posts List.
+	 * If the post object is the "front page," this will include the main dashboard link instead.
+	 *
+	 * @see https://developer.wordpress.org/reference/hooks/page_row_actions/
+	 * @see https://developer.wordpress.org/reference/hooks/post_row_actions/
+	 * @param [string[]] $actions  The list of actions to which this function will append
+	 * @param [WP_Post]  $post     The individual post object in the list
+	 * @return $actions            The filtered list of actions
+	 */
+	public function row_actions_add_parsely_link( $actions, $post ) {
+		if ( ! is_array( $actions ) ) {
+			return $actions;
+		}
+
+		if ( ! self::post_has_trackable_status( $post ) ) {
+			return $actions;
+		}
+
+		$options = $this->get_options();
+		$apikey = $options['apikey'];
+		$parsely_url = trailingslashit( 'https://dash.parsely.com/' . $apikey );
+
+		if (
+			'page' === get_option( 'show_on_front' ) &&
+			(int) get_option( 'page_on_front' ) === $post->ID
+		) {
+			// This is the "front page." Link to the dashboard.
+			$actions[ 'parsely_dash' ] = sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_url( $parsely_url ),
+				esc_html__( 'Parse.ly Dashboard', 'wp-parsely' )
+			);
+			return $actions;
+		}
+
+		$post_url = get_permalink( $post );
+		$parsely_url .=  'find?url=' . urlencode( $post_url );
```

Comment:
Does Parse.ly require `urlencode()`'d querystring, or `rawurlencode()`'d querystring? They do give slightly different results.

Reply:
I think we may want to switch to [`rawurlencode`](https://www.php.net/manual/en/function.rawurlencode.php) for the full [RFC3986](http://www.faqs.org/rfcs/rfc3986.html) treatment.  See also:
https://stackoverflow.com/a/996161

@hbbtstar -- do you have any insight into what the service expects?

Reply:
I went ahead and switched to `rawurlencode`. I think that's probably preferable, but we can change it back if needed.